shared/search/user-search/render.native.js

```diff
@@ -42,13 +45,13 @@ function KeybaseExtraInfo ({username, fullName, isFollowing, searchText}) {
   )
 }
 
-// TODO(MM) use serviceAvatar
-function ExternalExtraInfo ({serviceUsername, fullNameOnService, icon, searchText}) {
+function ExternalExtraInfo ({fullNameOnService, icon, serviceAvatar, serviceUsername, searchText}) {
```

Comment:
i honestly would rather go the other direction, having one component thats driven by props that draws itself correctly instead of 4 variants that are very similar. we already have too much duplication in this file (and also native is also super close and should be hoisted anyways)